storage/tokudb/tokudb_thread.h

```diff
@@ -372,28 +372,28 @@ inline void event_t::wait(void) {
     assert_debug(r == 0);
     return;
 }
-inline int event_t::wait(ulonglong microseconds) {
-    timespec waittime = time::offset_timespec(microseconds);
-    int r = pthread_mutex_timedlock(&_mutex, &waittime);
-    if (r == ETIMEDOUT) return ETIMEDOUT;
-    assert_debug(r == 0);
-    while (_signalled == false && _pulsed == false) {
-        r = pthread_cond_timedwait(&_cond, &_mutex, &waittime);
-        if (r == ETIMEDOUT) {
-            r = pthread_mutex_unlock(&_mutex);
-            assert_debug(r == 0);
-            return ETIMEDOUT;
-        }
-        assert_debug(r == 0);
-    }
-    if (_manual_reset == false)
-        _signalled = false;
-    if (_pulsed)
-        _pulsed = false;
-    r = pthread_mutex_unlock(&_mutex);
-    assert_debug(r == 0);
-    return 0;
-}
+//inline int event_t::wait(ulonglong microseconds) {
+//    timespec waittime = time::offset_timespec(microseconds);
+//    int r = pthread_mutex_trylock(&_mutex, &waittime);
+//    if (r == ETIMEDOUT) return ETIMEDOUT;
+//    assert_debug(r == 0);
+//    while (_signalled == false && _pulsed == false) {
+//        r = pthread_cond_timedwait(&_cond, &_mutex, &waittime);
+//        if (r == ETIMEDOUT) {
+//            r = pthread_mutex_unlock(&_mutex);
+//            assert_debug(r == 0);
+//            return ETIMEDOUT;
+//        }
+//        assert_debug(r == 0);
+//    }
+//    if (_manual_reset == false)
+//        _signalled = false;
+//    if (_pulsed)
+//        _pulsed = false;
+//    r = pthread_mutex_unlock(&_mutex);
+//    assert_debug(r == 0);
+//    return 0;
+//}
```

Comment:
suggest deleting functions rather than commenting them out.

Reply:
Sure, I wasn't sure if it was acceptable to delete, so I just commented for discussion. Will remove in the split patches.